docs/dcp2_system_design.rst

```diff
@@ -1824,3 +1824,61 @@ The ``supplementary_file.provenance.submitter_id`` field is set to
 ``c9efbb15-c50c-5796-8d15-35e9e1219dc5``. This UUIDv5 was generated using the
 same namespace UUID as for `Contributor-generated matrices (CGMs)`_ and the
 name ``dcp1 matrix service``.
+
+
+
+
+Access controls
+===============
+
+The DCP/2 manages access to the (meta)data that it hosts. When a *principal*
+(user, network, process, or application) that is not part of the DCP/2
+requests to retrieve a *resource* (metadata entity or data file) from a
+component of the DCP/2, the following rules apply:
+
+1) If the TDR snapshot containing the requested resource is marked public,
+   the request will be satisfied, independently of whether the principal's
+   identity is evident from the request or not.
+
+2) Otherwise, if the request is for metadata in the ``project`` entity, and
+   the request is made to the Azul REST API, the request will be satisfied.
+
+3) Otherwise, if the request is for a `contributor-generated project-level
+   matrix data file <Contributor-generated matrices (CGMs)_>`_, and the request
+   is made to the Azul REST API, the request will be satisfied.
+
+4) Otherwise, if the request is accompanied by a valid OAuth 2.0 token, and
+   the identity associated with that token is registered with Terra's SAM and
+   the identified principal has been granted access to the TDR snapshot that
+   contains the requested resource, the request will be satisfied.
```

Comment:
Not super familiar with how these auth requests are made, so I just want to double check - Is this access recorded anywhere? e.g. in a scenario where someone with the proper permissions asks us for a list of people who accessed the project's data, will we have this list?

Reply:
We will. Logging and auditing are required by NIST-800 which the DB and Azul teams are working towards being certifiably compliant with. This will enable us to interconnect with the Terra ecosystem which already is compliant, I believe with a superset of that. I think it's better to not duplicate compliance requirements here.